docs/1.0. Overview.md

```diff
@@ -21,18 +21,12 @@ The API is "Northbound" of the Network Controller. The purpose of this API is to
 * Assure bandwidth for these media flows
 * Ensure network security by only allowing authorized senders, receivers and flows.
 
-The resources that are managed by the AMWA IS-06 NMOS Network Control API are as follows:
```

Comment:
Sentence to add:

Conventions used in the Network Control API that are common with the specification of AMWA IS-04 and IS-05 are covered in the next section, [APIs](2.0.%20APIs.md).